static/translations/messages_ar.json

```diff
@@ -6,6 +6,9 @@
   "Private messages": "رسائل خاصة",
   "Starred": "مميزة بنجمة",
   "Mentions": "الإشارات",
+  "Message": "Message",
+  "Send": "إرسال",
+  "Please choose recipients to share with": "Please choose recipients to share with",
```

Comment:
Note: The "Portuguese (Brazil)" message, added in messages_en.json
    in this series, is added to the other languages in this commit,
    instead of in a separate "Sync recently-added message strings"
    commit. That's because I ran `tools/tx-sync` twice after that string
    was added, and I kept the result from the second time. This was a
    small mistake that I don't expect to cause real problems, but it
    does explain what happened with that string.

Yeah, and it looks like the same thing happened with a number of these other strings.

Nothing all that bad happens -- nothing at all goes wrong at runtime. The only effect is that it makes it hard to spot the actual new translations in the history. (You can get some idea by looking at the diffstat and seeing that some languages have more edits than others; but then there's also cases like this one where a new string already has a translation.)

Definitely not worth trying to go back and pick apart the changes into two commits. But that's the reason why I developed a habit of doing first a pull for new translations on existing strings, committing that, then a push + pull to sync translations across. And then codifying that habit into the tools/tx-sync script :slightly_smiling_face: 

Basically, the Transifex push that happens in the last step of tools/tx-sync has side effects -- it pushes strings to Transifex, and so can immediately change what the next pull from Transifex will get -- so you always want to push (in Git) the result of the first step of tools/tx-sync, where it pulled translations down from Transifex.